data/AddonsManagerMod.js

```diff
@@ -121,22 +117,59 @@ ACRController.makeButtonUI = function(addonReport)
         label.setAttribute("value", "Compatibility Problems"); // TODO l10n
         hbox.appendChild(label);
 
-        return hbox;
+        hbox_outer.appendChild(hbox);
+    } else {
+        var button = document.createElement("button");
+        button.setAttribute("label", "Report Issue");
+        button.setAttribute("class", "anon-control");
+
+        //button.addEventListener("click", function() { ACRController.openSendReportDialog(addonReport); }, true);
+        button.addEventListener("click", function() {
+            //ACRController.openSendReportDialog(addonReport);
+            self.port.emit("acr_open_submit_report_dialog", addonReport);
+        }, true);
+        hbox_outer.appendChild(button);
     }
+    return hbox_outer;
+}
 
-    var button = document.createElement("button");
-    button.setAttribute("label", "Report Issue"); // TODO l10n
-    button.setAttribute("class", "anon-control");
-    button.setAttribute("owner", "acr");
+ACRController.makeE10sInfo = function(addonReport) {
+    var hbox = document.createElement("hbox");
```

Comment:
Fair enough.

- tofumatt

On 23 August 2016 at 20:04:35, Andreas Wagner (notifications@github.com)
wrote:

> In data/AddonsManagerMod.js
> <https://github.com/mozilla/addon-compatibility-reporter/pull/54#discussion_r75930649>
> :
>
> >
> > -    var button = document.createElement("button");
> > -    button.setAttribute("label", "Report Issue"); // TODO l10n
> > -    button.setAttribute("class", "anon-control");
> > -    button.setAttribute("owner", "acr");
> > +ACRController.makeE10sInfo = function(addonReport) {
> > +    var hbox = document.createElement("hbox");
>
> Let's not do this. This is a XUL document and jQuery breaks in some cases,
> believe me I tried...
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/addon-compatibility-reporter/pull/54/files/559185eb636f9004e09bf4235a959d4a9e321497#r75930649>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFi98E2rQt5RevIzSUQhMr-q3Iw7nOrks5qi0RDgaJpZM4Jnslc>
> .
>